download.md

```diff
@@ -1,90 +1,224 @@
-| [Named-Entity Recognition](https://github.com/RubenVerborgh/Refine-NER-Extension) | With this extension, you can enrich text fields right from your workspace. It supports AlchemyAPI, DBpedia Lookup and Zemanta.  - see also their [documentation](http://freeyourmetadata.org/named-entity-extraction/)  |  Ruben Verborgh (Free Your Metadata) | 
-| [Stats extension for Google Refine 2.1](https://github.com/newsapps/refine-stats)  | A extension for OpenRefine to compute elementary statistics on a column data. | the Chicago Tribune | 
-| [Stats extension for Google Refine 2.5+](https://github.com/sparkica/refine-stats) | A extension for OpenRefine to compute elementary statistics on a column data.  | sparkica  | 
-| [extraCTU-plugin](https://github.com/giTorto/geoXtension) | to extract E-mails, Telephone numbers, Urls and the Identification Numbers from a messy text | giTorto |
-| [geoXtension](https://github.com/giTorto/geoXtension) | geoXtension is based on gdal, proj and geos libraries. Adding this extension to OpenRefine takes a lot of time, usage of the ready-made docker is recommended |  giTorto  | 
-| [D2Refine](https://github.com/caCDE-QA/D2Refine) | A clinical study metadata harmonization and validation workbench to 1) convert restrictions (specified in a spreadsheet like interface) into a clinical model; and 2) provide metadata reconciliation services leveraging the Common Terminology Services 2 (CTS2) |  Deepak K. Sharma https://github.com/dksharma | 
-
-## Reconciliation Services
-
-OpenRefine can connect to several reconciliation services. Please read our wiki for a [list of them](https://github.com/OpenRefine/OpenRefine/wiki/Reconcilable-Data-Sources).
-
-## Client Libraries
-
-You can automate some OpenRefine operations using one of the existing libraries. Those libraries are using the [OpenRefine API](https://github.com/OpenRefine/OpenRefine/wiki/OpenRefine-API).
-* [python](https://github.com/PaulMakepeace/refine-client-py/)
-  * [fork with extended CLI](https://github.com/felixlohmeier/openrefine-client)
-* An other [python](https://github.com/maxogden/refine-python) library
-* [ruby](https://github.com/maxogden/refine-ruby)
-* [javascript - nodejs](https://github.com/pm5/node-openrefine)
-* [R](https://cran.r-project.org/web/packages/rrefine/index.html)
-* [php](https://github.com/keboola/openrefine-php-client)
+
+
+    
+<div id="content">
+  <p>You will find on this page a list of OpenRefine distributions and extensions available for download. Are we missing something? Want to fix a typo? You can submit changes (pull request) <a href="https://github.com/OpenRefine/openrefine.github.com/blob/master/download.md">from here</a>.</p>
```

Comment:
- [ ] Let's move "Are we missing something" into the Sidebar as a Site Feedback link called "Website feedback".

- [ ] Let's also add a "Contact Us" link into the Sidebar as well that points to this link https://github.com/OpenRefine/OpenRefine#contact-us